db/create_analytics_views.sql

```diff
@@ -166,7 +166,7 @@ CREATE OR REPLACE VIEW analytics.stimulus_triages AS
     FROM public.stimulus_triages;
 
 CREATE OR REPLACE VIEW analytics.system_notes AS
-    SELECT id, client_id, created_at, updated_at, user_id
+    SELECT id, client_id, created_at, type, updated_at, user_id
```

Comment:
I just noticed that should probably change client_id too, since it no longer exists (replaced by record_id & record_type)! If you have any ideas for how to keep these analytics better in sync with migrations, let me know.

Reply:
That's the access_logs table!